rust/migrate-wicked/src/wireless.rs

```diff
@@ -59,9 +61,16 @@ impl From<&WickedWirelessMode> for model::WirelessMode {
     }
 }
 
-#[derive(Debug, PartialEq, Serialize, Deserialize)]
+#[derive(Debug, PartialEq, Serialize, Deserialize, Default)]
 pub struct WpaPsk {
     pub passphrase: String,
+    #[serde(rename = "auth-proto")]
+    pub auth_proto: Option<EapAuthProto>,
+    #[serde(rename = "pairwise-cipher")]
+    pub pairwise_cipher: Option<EapPairwiseCipher>,
```

Comment:
`pairwise-cipher` and `group-cipher` are defined as
```
<group-cipher type="builtin-wireless-cipher-mask"/>
<pairwise-cipher type="builtin-wireless-cipher-mask"/>
```
Which result in values like: `<pairwise-cipher>CCMP,TKIP</pairwise-cipher>`

With this we need to make them a `vec`.